src/test/java/Problem2Test.java

```diff
@@ -7,13 +7,29 @@
     public void testRemoveDuplicates() {
         int[][] inputs = {
                 // DO NOT REMOVE OR CHANGE THE GIVEN TEST CASES!
-                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6}
-                // add more test cases
+                {0, 0, 1, 1, 1, 2, 2, 4, 4, 6},
+                {},
+                {1,2,3,4,5},
+                {0,0,0,0},
+                {1,2,5,5,5,5},
+                {-5,-5,-3,-2,-1},
+                {0,0,1,1,1,2},
+                {1,1,1,2,3},
+                {1,2,2,2,3},
+                {1,2,3,3,3},
```

Comment:
format doesn't look right. use the code format from Intellij.

there's a format code option when make commit. it's handy for a final formatting right before code is checked in.